cmd/podman/networks/list.go

```diff
@@ -38,6 +38,7 @@ var (
 	networkListOptions entities.NetworkListOptions
 	filters            []string
 	noTrunc            bool
+	defaultListRow     = "{{.Name}}\t{{.Version}}\t{{.Plugins}}\n"
```

Comment:
```suggestion
	defaultListRow     = "{{.Id}}\t{{.Name}}\t{{.Version}}\t{{.Plugins}}\n"
```

This is the docker output today...

```shell
$ docker network ls
NETWORK ID          NAME                DRIVER          SCOPE
7fca4eb8c647        bridge              bridge          local
```

Reply:
added, tried .Id but it's set for only .ID